FWCore/Integration/test/ESTestAnalyzers.cc

```diff
@@ -213,10 +214,28 @@ namespace edmtest {
                                         << ": Data values = " << dataJ->value();
   }
 
+  class ESTestAnalyzerL : public edm::stream::EDAnalyzer<> {
+  public:
+    explicit ESTestAnalyzerL(edm::ParameterSet const& iConfig)
+        : edToken_(consumes(iConfig.getParameter<edm::InputTag>("src"))), esToken_(esConsumes()) {}
+    void analyze(const edm::Event&, const edm::EventSetup&) override;
+
+  private:
+    edm::EDGetTokenT<IntProduct> edToken_;
+    edm::ESGetToken<ESTestDataJ, ESTestRecordJ> esToken_;
+  };
+
+  void ESTestAnalyzerL::analyze(edm::Event const& ev, edm::EventSetup const& es) {
+    auto const& dataJ = es.getData(esToken_);
```

Comment:
The idea is to test on existence not on content?

Reply:
Right, the de-reference of the `ESHandle` is the important part. I change the following printout to Info or Debug since it is not really important (mostly a leftover from copy-paste from `ESTestAnalyzerJ`).